koku/api/metrics/models.py

```diff
@@ -56,3 +56,5 @@ class Meta:
     label_measurement = models.CharField(max_length=256, null=False)
 
     label_measurement_unit = models.CharField(max_length=64, null=False)
+
+    default_cost_type = models.TextField(null=True)
```

Comment:
Is the new strategy to **not** add the choices at the model definition?
Also: should a default be set here to match the default used below in the serializer?